### DISCUSSION TITLE
get coordinates from an INSERT

### DISCUSSION THREAD

**Discussion by aminreguig** [mozman/ezdxf#837] [General]
I'm  trying to get INSERT coordinates with  :

insert_entities = self.dxf_file.modelspace().query('INSERT[layer=="%s"]' %  layer )
for entity in insert_entities :
     for attr in entity.attribs:
            print( attr.dxf.tag)
            print(attr.dxf.text)

This code doesn't return all the infermations (ps : I can see coordinates by visualizing the dxf file in a text editor)

**Comment by mozman** [comment]
Please read the documention: https://ezdxf.mozman.at/docs/usage_for_beginners.html
